bookShop/src/app/shared/shared.module.ts

```diff
@@ -0,0 +1,20 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { FormsModule } from '@angular/forms';
+import { BrowserModule } from '@angular/platform-browser';
+import { OrdersModule } from './orders.module';
+
+@NgModule({
+  declarations: [],
+  imports: [
+    BrowserModule,
+    FormsModule,
+  ],
+  exports: [
+      CommonModule,
+      OrdersModule
+  ],
+  providers: [],
+  bootstrap: []
```

Comment:
You don't need a `bootstrap` module. It uses only in `AppModule`

Reply:
Corrected